### ISSUE TITLE
[duvida

### ISSUE THREAD

**Issue by gabrielbahniuk** [da2k/curso-javascript-ninja#4872]
Escreva aqui sua dúvida

@fdaciuk

**Comment by fdaciuk** [comment]
Oi @gabrielbahniuk! Nesse caso, você precisaria exportar as funções que você pretende testar.

Se for rodar a suíte de testes direto no browser, você pode apenas "pendurar" no objeto `window` o que precisa exportar.

Agora, se for rodar os testes em ambiente Node, e precisar ter acesso às funções à partir do terminal, então você precisa exportar em um formato que o Node entenda, saca? =)

O formato "ideal" para funcionar tanto em um como em outro ambiente, seria usar UMD.

Eu escrevi alguns artigos sobre alguns sistemas de módulos, e como você consegue implementá-los. Dá uma olhada nesses links:

https://blog.da2k.com.br/2015/01/03/como-criar-componentes-js-usando-modulos-amd-commonjs-e-umd-parte-1-3/

https://blog.da2k.com.br/2015/01/04/como-criar-componentes-js-usando-modulos-amd-commonjs-e-umd-parte-2-3/

https://blog.da2k.com.br/2015/01/05/como-criar-componentes-js-usando-modulos-amd-commonjs-e-umd-parte-3-3/

**Comment by fdaciuk** [comment]
Exatamente! A ideia do UMD é simplesmente colocar o que vai ser exportado em uma função, e chamar conforme o ambiente ou o sistema de módulos que você estiver usando =)
